pages/docs/tutorials/getting-started/hello-world.md

```diff
@@ -0,0 +1,162 @@
+        payload:
+          type: string
+          pattern: '^hello .+$'
+operations:
+  receiveHello:
+    action: 'receive'
+    channel:
+      $ref: '#/channels/hello'`}
+</CodeBlock>
+Let's get into the details of this sample specification:
+
+<CodeBlock highlightedLines={[1]}>
+{`asyncapi: 3.0.0
+info:
+  title: Hello world application
+  version: '0.1.0'
+channels:
+  hello:
+    address: 'hello'
+    messages:
+      sayHelloMessage:
+        payload:
+          type: string
+          pattern: '^hello .+$'
+operations:
+  receiveHello:
+    action: 'receive'
+    channel:
+      $ref: '#/channels/hello'`}
+</CodeBlock>
+The first line of the specification starts with the document type `asyncapi` and the version (3.0.0). This line doesn't have to be the first one, but it's a recommended practice.
+<CodeBlock highlightedLines={[2,3,4]}>
+{`asyncapi: 3.0.0
+info:
+  title: Hello world application
+  version: '0.1.0'
+channels:
+  hello:
+    address: 'hello'
+    messages:
+      sayHelloMessage:
+        payload:
+          type: string
+          pattern: '^hello .+$'
+operations:
+  receiveHello:
+    action: 'receive'
+    channel:
+      $ref: '#/channels/hello'`}
+</CodeBlock>
+The `info` object contains the minimum required information about the application. It contains the `title`, which is a memorable name for the API, and the `version`. While it's not mandatory, it's strongly recommended to change the version whenever you make changes to the API.
+
+<CodeBlock highlightedLines={[5,6,7,8,9,10,11,12]}>
+{`asyncapi: 3.0.0
+info:
+  title: Hello world application
+  version: '0.1.0'
+channels:
+  hello:
+    address: 'hello'
+    messages:
+      sayHelloMessage:
+        payload:
+          type: string
+          pattern: '^hello .+$'
+operations:
+  receiveHello:
+    action: 'receive'
+    channel:
+      $ref: '#/channels/hello'`}
+</CodeBlock>
+
+The `channels` section of the specification houses all of the mediums where messages flow through. For example, some systems use `topic`, `event name` or `routing key`. Different kinds of information flow through each channel similar to the analogy of TV channels.
+
+In this example, you only have one channel called `hello`. The sample application subscribes to this channel to receive `hello {name}` messages.
+
+<CodeBlock highlightedLines={[6,7,8,9]}>
+{`asyncapi: 3.0.0
+info:
+  title: Hello world application
+  version: '0.1.0'
+channels:
+  hello:
+    address: 'hello'
+    messages:
+      sayHelloMessage:
+        payload:
+          type: string
+          pattern: '^hello .+$'
+operations:
+  receiveHello:
+    action: 'receive'
+    channel:
+      $ref: '#/channels/hello'`}
+</CodeBlock>
+
+You can read the highlighted lines as:
+> This is the `payload` of the `message` that the `Hello world application` is subscribed to. You can pass the value of `address` to the `messages` to the `hello` channel and the `Hello world application` will receive it through `operations` object.
```

Comment:
This single line is really confusing 😅 
- `This is the payload of the message that the Hello world application is subscribed to` you don't subscribe to a message, you subscribe to a topic where you receive certain messages.
- `You can pass the value of address to the messages to the hello channel` too many `to the` creates confusion for me.
- `through operations object` makes no sense to me

Reply:
sure, i will update it